_posts/2017-01-15-webapps.markdown

```diff
@@ -205,6 +200,12 @@ Start by creating a directory called ```flask-app``` where we'll create the foll
 
 Make sure to ```cd flask-app``` before you start creating the files, because you don't want to start adding a whole bunch of other random files to your image.
 
+You will also need a text editor. We recommend vim, which will already be installed in the terminal. However if you wanted, for example, emacs, you would install it using the Alpine package manager:
+```
```

Comment:
You can add .term1 here if you want and emacs will get installed in the instance. There's no need to do `update` also, you can just make `apk add --no-cache emacs`